.github/workflows/ci.yml

```diff
@@ -48,7 +48,6 @@ jobs:
         node:
           - 18
           - 20
-          - 21
           - 22
```

Comment:
It looks like a good opportunity to add 23.x to the matrix, wdyt?
```suggestion
          - 22
          - 23
```

Reply:
Why not update better-sqlite in the same PR?

Reply:
The alternative would be one PR to
- drop Node 21
- upgrade better-sqlite2
- add Node 23
all together

Reply:
OK, I will add the additional commits to this PR, if that is what you prefer.

Reply:
Done

Reply:
Node.js 23 is now added again